osidb/management/commands/collectfeed.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import traceback
+
+from django.core.management.base import BaseCommand
+
+from collectors.feed.collectors import get_snippets
+from collectors.feed.collectors import log as feed_log
+
+
+class Command(BaseCommand):
+    help = "Runs feed collector to create new Flaw drafts in OSIDB"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--dry",
+            action="store_true",
+            help="Dry run: Do not memorize already-fetched feed entries and do not create new Flaw drafts. You can set --verbosity=2 or 3 to observe the dry run.",
+        )
+
+    def handle(self, *args, **options):
+        verbosity = options["verbosity"]
+        old_level = logging.root.manager.root.level
+
+        # Enhance logging only for this command
+        custom_level = False
+        if verbosity in [2, 3]:
+            custom_level = {2: logging.INFO, 3: logging.DEBUG}[verbosity]
+            logging.basicConfig(
+                level=custom_level,
+            )
+
+        dry_run = options["dry"]  # TODO: Eliminate dry run?
+        self.stdout.write("Dry run")
+        try:
+            feed_log.setLevel(custom_level)
+            get_snippets(dry_run=dry_run)
+        except Exception as e:
+            self.stdout.write(self.style.ERROR("FAIL"))
+            if options["verbosity"] == 2:
+                self.stdout.write(f"Exception: {e}")
+            elif options["verbosity"] == 3:
+                self.stdout.write(traceback.format_exc())
+        else:
+            self.stdout.write(self.style.SUCCESS("OK"))
+
+        # Revert the enhanced logging
+        if verbosity > 1:
+            logging.basicConfig(
+                level=old_level,
+            )
+            feed_log.setLevel(old_level)
```

Comment:
see https://docs.python.org/3/library/logging.html#logging.disable which is how `syncflaws` does it

Reply:
Thanks. Used this feedback to make the logging work mostly the same as with syncflaws: The commandline util works the same and the collector logs roughly the same information and the logs can therefore be accessed roughly the same way. It's then up to the configuration whether the logs are visible.